lib/common.sh

```diff
@@ -142,8 +142,12 @@ export NUM_OF_WORKER_REPLICAS="${NUM_OF_WORKER_REPLICAS:-"1"}"
 export VM_EXTRADISKS=${VM_EXTRADISKS:-"false"}
 export NODE_DRAIN_TIMEOUT=${NODE_DRAIN_TIMEOUT:-"0s"}
 
+# Registry for pulling images
+export OVERRIDE_DOCKER_REGISTRY_1=${OVERRIDE_DOCKER_REGISTRY_1:-"docker.io"}
+export OVERRIDE_DOCKER_REGISTRY_2=${OVERRIDE_DOCKER_REGISTRY_2:-"registry.hub.docker.com"}
+
 # Docker registry for local images
-export DOCKER_REGISTRY_IMAGE=${DOCKER_REGISTRY_IMAGE:-"docker.io/registry:latest"}
+export DOCKER_REGISTRY_IMAGE=${DOCKER_REGISTRY_IMAGE:-"${OVERRIDE_DOCKER_REGISTRY_1}/library/registry:latest"}
```

Comment:
I think `/library/` is not needed here. It will work without it.

Reply:
fixed